### ISSUE TITLE
Potential contributions for Lecture 5

### ISSUE THREAD

**Issue by rachtsingh** [harvard-ml-courses/cs281-scribe#55]
- solve the posterior predictive exercise in 5.5 (use the `exercise` environment, as described [here](https://github.com/harvard-ml-courses/cs281-scribe/blob/master/scribe_notes.md#exercises)
- replace all of the `log`s with `\log`s
- replace the softmax plot. I think it's useless as is.
- in section 5.9, add an explanation of the logsumexp trick (see [here](https://hips.seas.harvard.edu/blog/2013/01/09/computing-log-sum-exp/) for intuition). It shouldn't be very long. 
- as an exercise, write the PyTorch code for running a logistic regression (there's a million tutorials for this online). 

Meeting the points counts as fulfilling the scribing requirements. Please respond if you're taking - that way we can avoid duplicating work.

**Comment by parthmehta117** [comment]
I'll work on this.
